benchmarks/.bundle/config

```diff
@@ -0,0 +1,3 @@
+---
+BUNDLE_RETRY: "3"
```

Comment:
Where are these used?

Reply:
Honestly I didn't realize this file existed or was committed. Cautiously removing!